tests/sharingTest.js

```diff
@@ -123,6 +122,8 @@ describe('Main: Currently testing file/folder sharing,', function () {
       ocs.appendElement('meta', '', (meta) => {
         ocsMeta(meta, 'ok', '100', MatchersV3.regex('(OK)?', ''))
       }).appendElement('data', '', data => {
+        // following elements are matched serially
+        // so any changes in the order (from the server) breaks the tests
```

Comment:
some tests could pass in oCIS but due to this the tests fail.
I couldn't find the way to match them in any order.

Reply:
the test is trying to match shares
- user share
- group share
- public share

Reply:
please open an issue and add to out ToDo list, we can work on it later

Reply:
Raised an issue for this: https://github.com/owncloud/owncloud-sdk/issues/1094